.gitattributes

```
@@ -0,0 +1,10 @@
+/.github            export-ignore
+/tests              export-ignore
+.gitattributes      export-ignore
```

```suggestion
.gitattributes      export-ignore
.allowed-licenses.php export-ignore
```

will need indentation adjusting 😭